launch/lifelong_launch.py

```diff
@@ -1,17 +1,51 @@
-from launch import LaunchDescription
-import launch_ros.actions
 from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import EmitEvent, LogInfo, RegisterEventHandler
+from launch.events import matches_action
+from launch_ros.actions import LifecycleNode
+from launch_ros.event_handlers import OnStateTransition
+from launch_ros.events.lifecycle import ChangeState
+from lifecycle_msgs.msg import Transition
 
 
```

Comment:
Add an launch parameterization for if the lifecycle transition should autostart or not, defaulting to true. That way in Nav2 we can override that to false so that the lifecycle manager can handle the node states itself

Reply:
[444c5d2cec35b159d81fb7522127e23065483399]

- remove namespace `/` (ref. <https://github.com/SteveMacenski/slam_toolbox/pull/561#discussion_r1091045236> )
- add `autostart` launch parameter